planning/behavior_path_planner/src/util/drivable_area_expansion/footprints.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "behavior_path_planner/util/drivable_area_expansion/footprints.hpp"
+
+#include "behavior_path_planner/util/drivable_area_expansion/parameters.hpp"
+
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
+
+#include <boost/assign.hpp>
+#include <boost/geometry.hpp>
+
+#include <tf2/utils.h>
+
+namespace drivable_area_expansion
+{
+polygon_t rotatePolygon(const polygon_t & polygon, const double angle)
```

Comment:
FYI: The similar function exists here. The argument type is not the same though.
You can move your util function used in other package here. For me, it's not necessary to do that in this PR.
https://github.com/autowarefoundation/autoware.universe/blob/ded1d94524e673d871f60b7fd4268d7a417ce304/common/tier4_autoware_utils/src/geometry/boost_polygon_utils.cpp#L93-L94